test_scripts/RC/CLIMATE_RADIO/Capabilities/008_Default_capabilities_if_HMI_doesnt_respond_to_RC_IsReady_and_doesnt_respond_to_RC_GetCapabilities_RADIO.lua

```diff
@@ -34,12 +34,17 @@ local function getHMIParams()
   return params
 end
 
+local function start(pHMIParams)
+  commonRC.start(pHMIParams)
+  commonRC.wait(12000)
```

Comment:
why do we have to wait 12seconds here? To ensure the GetCapabilities requests time out?

Reply:
@iCollin Yes, you are right. SDL waits responses from HMI and after timeout loads default capabilities from the file. 
Without this delay we face timeout on RAI request, because SDL postpones RAI requests until HMI capabilities is set (received from HMI or load from file)

Reply:
Do you think this would be a problem in production? Will real world apps get RAI timeout connecting to a car? Should Core not time out RAI while waiting for getcapabilities responses?

Reply:
@iCollin I am not exactly familiarized with SDL SDK for Android and iOs but I think it will not be a problem if mobile aplication do wait enough for RAI response (SDL should respond on RAI during time equals to 2 base timeouts for request)

> Should Core not time out RAI while waiting for getcapabilities responses?
Current proposal adds suspension of RAIs processing untill GetCapabilities responses received.

Reply:
Yes, the wait shouldn't be necessary here, Core should not be timing out RAI before GetCapabilities responses are received per the proposal. I will request this change on the Core PR and once that change is made we can remove the wait here.